src/openforms/registrations/contrib/json_dump/plugin.py

```diff
@@ -0,0 +1,142 @@
+import base64
+
+from django.utils.translation import gettext_lazy as _
+
+from zgw_consumers.client import build_client
+
+from openforms.formio.typing import Component
+from openforms.forms.utils import to_json
```

Comment:
this is a private utility because it's not exported by `openforms.forms.service`.